### PR TITLE
Checklist: Translate the onboarding checklist items

### PR DESCRIPTION
**Pull request by taggon**
Since we decided to roll out the onboarding checklist to 100% users, the checklist items should be translated before #25929 is merged.

The duration has now become "minute" or "minutes".

## Screens

jp
minutes

# How to test

1. Create a free website with a non-English user account.
2. You should be able to see the checklist items translated in your language.

### CODE REVIEW

File: client/my-sites/checklist/onboardingChecklist.js [hunk 1]
```diff
@@ -5,81 +5,89 @@
...
 		image: '/calypso/images/stats/tasks/upload-profile-picture.svg',
 		tour: 'checklistUserAvatar',
 	},
 	blogname_set: {
-		title: 'Give your site a name',
-		description: 'Give your site a descriptive name to entice visitors.',
-		duration: '1 min',
-		completedTitle: 'You updated your site title',
-		completedButtonText: 'Edit',
+		title: i18n.translate( 'Give your site a name' ),
```
Comment (a8ci18n) [inline client/my-sites/checklist/onboardingChecklist.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Give your new site a name.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by ockham** [comment]
FYI @sirreal

**Comment by sirreal** [comment]
Shall we just roll this into #25956? There are going to by significant conflicts.

File: client/my-sites/checklist/onboardingChecklist.js [hunk 2]
```diff
@@ -5,81 +5,89 @@
...
-		description:
-			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.',
-		duration: '2 mins',
-		completedTitle: 'You uploaded a profile picture',
-		completedButtonText: 'Change',
+		title: i18n.translate( 'Upload your profile picture' ),
+		description: i18n.translate(
+			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.'
+		),
+		duration: i18n.translate( '%d min', '%d mins', { count: 2, args: [ 2 ] } ),
```
Comment (sirreal) [inline client/my-sites/checklist/onboardingChecklist.js]: For a static string like `2 mins`, do we need to use the number format, or just `translate( "2 mins" )`?

**Comment by ockham** [comment]
I think we shouldn't, b/c separation on concerns TBH :slightly_frowning_face:

**Comment by ockham** [comment]
(I don't mind doing the rebase)

File: client/my-sites/checklist/onboardingChecklist.js (hunk 2 above)
Comment (ockham) [reply client/my-sites/checklist/onboardingChecklist.js]: I think the idea is to 'translate `%d min` (and `%d mins`) once, re-use for all numbers'.

**Comment by taggon** [comment]
I wasn't aware of #25956. I'll update this PR to use `translate` instead of `i18n.translate` to sync with #25956, so it would be a bit easier to rebase.

File: client/my-sites/checklist/onboardingChecklist.js [hunk 3]
```diff
@@ -5,81 +5,87 @@
...
 		image: '/calypso/images/stats/tasks/upload-profile-picture.svg',
 		tour: 'checklistUserAvatar',
 	},
 	blogname_set: {
-		title: 'Give your site a name',
-		description: 'Give your site a descriptive name to entice visitors.',
-		duration: '1 min',
-		completedTitle: 'You updated your site title',
-		completedButtonText: 'Edit',
+		title: translate( 'Give your site a name' ),
```
Comment (a8ci18n) [inline client/my-sites/checklist/onboardingChecklist.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Give your new site a name.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/checklist/onboardingChecklist.js [hunk 4]
```diff
@@ -5,81 +5,87 @@
...
 		image: '/calypso/images/stats/tasks/upload-profile-picture.svg',
 		tour: 'checklistUserAvatar',
 	},
 	blogname_set: {
-		title: 'Give your site a name',
-		description: 'Give your site a descriptive name to entice visitors.',
-		duration: '1 min',
-		completedTitle: 'You updated your site title',
-		completedButtonText: 'Edit',
+		title: translate( 'Give your new site a name' ),
```
Comment (a8ci18n) [inline client/my-sites/checklist/onboardingChecklist.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Give your new site a name.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by ockham** [comment]
Rebased.

**Comment by sirreal** [comment]
I noticed we were using `3 min` –note no "s"– for plural (this looks fine to me as a native speaker). I removed them from the plurals in cd3e161

Then I noticed we were being inconsistent. One checklist used the plural "3 mins" and the other singular "3 min".

What should we do for the plurals of "min" here? @Automattic/editorial 

Live examples:

2 mins:
2mins

3 min:
3min

See a full checklist at `https://wordpress.com/checklist/MY-SITE-SLUG`

**Comment by benhuberman** [comment]
To be honest, I can't think of any compelling reason not to just spell out "minutes" here -- it just makes it easy to parse and to stay consistent, and (correct me if I'm wrong) those extra 3-4 characters aren't going to cause any issue design/layout-wise.

**Comment by sirreal** [comment]
Thanks @benhuberman, sounds good to me. I'll make the change cc: @joanrho

**Comment by mattwiebe** [comment]
image

Just did a quick test: a very long translation will be a bit ugly but "minutes" is just fine on a 320px wide screen (and even our notorious German friend is actually just *minuten*)

**Comment by sirreal** [comment]
Everything's translated, let's get this shipped!

**Pull request merged**